workers/generic-worker/run_task_as_current_user_windows.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"syscall"
+
+	"github.com/taskcluster/taskcluster/v80/workers/generic-worker/process"
+)
+
+func (r *RunTaskAsCurrentUserTask) resetPlatformData() {
+	r.task.pd = &process.PlatformData{
+		LoginInfo: &process.LoginInfo{},
+	}
+	for _, c := range r.task.Commands {
+		var token syscall.Token
+		c.SysProcAttr.Token = token
```

Comment:
can this be done in one line, with e.g. `c.SysProcAttr.Token = syscall.Token(0)`?